### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio.

### ISSUE THREAD

**Issue by var-bin** [Varbintech/next.varbintechcom#32]
Low-contrast text is difficult or impossible for many users to read. [Learn how to provide sufficient color contrast](https://dequeuniversity.com/rules/axe/4.6/color-contrast).

Failing Elements
--
MONTHY <br>`<p class="MuiTypography-root MuiTypography-body1 plan-title mui-style-1kixv5c">
MONTHY $4,999/m NO HONEYPOT Get it now Have a look our previous projects Wha…<div flag="MONTH" class="mui-style-rb0go8">`
BEST CHOICE <br>`<div class="mui-style-r0h4m4">`
QUARTERLY <br>`<p class="MuiTypography-root MuiTypography-body1 plan-title mui-style-1kixv5c">`
BEST CHOICE QUARTERLY $4,499/m YOU CAN SAVE $500 Get it now Have a look our… <br>`<div flag="QUARTER" class="mui-style-1gxtypl">`
YEARLY <br>`<p class="MuiTypography-root MuiTypography-body1 plan-title mui-style-1kixv5c">`
YEARLY $3,999/m YOU CAN SAVE $1000 Get it now Have a look our previous proje… <br>`<div flag="YEAR" class="mui-style-t0enyy">`

![13-26-24-ussnt](https://user-images.githubusercontent.com/3194163/228208961-3c237fb3-70ae-45d3-9bf0-8f7e5b76bf26.jpg)
![13-27-31-3x8nk](https://user-images.githubusercontent.com/3194163/228208977-77e9e9d9-6e9f-4bb2-9a57-3f85408f119a.jpg)
